include/seqan3/range/views/kmer_hash.hpp

```diff
@@ -511,26 +511,15 @@ class kmer_hash_view<urng_t>::shape_iterator
     }
 
     /*!\anchor shape_iterator_operator-difference
-     * \brief Return offset between this and remote iterator's position.
-     * \attention This function is only available if `it_t` models std::random_access_iterator.
-     */
-    difference_type operator-(shape_iterator const & lhs) const noexcept
```

Comment:
Didn't you just introduce this operator in the first commit ?

Reply:
In the first commit I introduced the friend `operator-`, here I remove the member `operator-`